package/msys/PKGBUILD

```diff
@@ -0,0 +1,67 @@
+# Author: williamjcm <w.jcm59@gmail.com>
+# Contributor: mosra <mosra@centrum.cz>
+# Based on the packages/archlinux/PKGBUILD file by mosra <mosra@centrum.cz> and the MSYS2/MinGW PKGBUILD templates
+
+_realname=magnum
+pkgbase=mingw-w64-${_realname}-git
+pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}-git"
+pkgver=dev
+pkgrel=1
+pkgdesc="C++11/C++14 graphics middleware for games and data visualization"
+arch=('any')
+url="https://magnum.graphics/"
+license=('MIT')
+# The corrade-git dependency corresponds to the package generated by Corrade's packages/msys/PKGBUILD file.
+depends=("${MINGW_PACKAGE_PREFIX}-corrade-git"
+         "${MINGW_PACKAGE_PREFIX}-openal"
+         "${MINGW_PACKAGE_PREFIX}-freeglut"
+         "${MINGW_PACKAGE_PREFIX}-SDL2"
+         "${MINGW_PACKAGE_PREFIX}-glfw"
+         "${MINGW_PACKAGE_PREFIX}-vulkan-loader")
+makedepends=("${MINGW_PACKAGE_PREFIX}-cmake" "${MINGW_PACKAGE_PREFIX}-ninja")
+options=('!strip')
+
+_rootdir=$startdir/../../
+
+build() {
+    mkdir -p "$_rootdir/build-${CARCH}"
+    cd "$_rootdir/build-${CARCH}"
+    
+    newcxxflags=$(echo $CXXFLAGS | sed s/-O.//g | sed s/-D_FORTIFY_SOURCE=.//g)
+    export CXXFLAGS="$newcxxflags"
+
+    MSYS2_ARG_CONV_EXCL="-DCMAKE_INSTALL_PREFIX=" \
+        ${MINGW_PREFIX}/bin/cmake .. \
+            -G'Ninja' \
+            -DCMAKE_BUILD_TYPE=Debug \
+            -DCMAKE_INSTALL_PREFIX=${MINGW_PREFIX} \
+            -DWITH_AUDIO=ON \
+            -DWITH_SHAPES=ON \
+            -DWITH_VK=ON \
+            -DWITH_GLFWAPPLICATION=ON \
+            -DWITH_GLUTAPPLICATION=ON \
```

Comment:
`GlutApplication` will be soon removed (it's now deprecated), so this doesn't need to be here. Thus also the `freeglut` dependency. I had it in the development PKGBUILD only because I still need to ensure it compiles correctly.

And the `Shapes` library is scheduled for removal as well, so no need for `WITH_SHAPES` either.

Reply:
Will do.